routes/userRoutes.js

```diff
@@ -2,7 +2,10 @@ const express = require('express');
 const router = express.Router();
 const userController = require('../controllers/userController');
 
-router.post('/register', userController.register);
+
+router.post('/verify', userController.resendVerifyToken);
 router.get('/verify/:verificationToken', userController.verifyEmail);
+router.post('/resend-verification-email', userController.resendVerificationEmail);
```

Comment:
Tego endpointu  nie potrzebujesz, bo ponowna wysyłka linku weryfikującego powinna się odbyć na /verify, więc jak user wejdzie na /verify to tam trzeba obsłużyć wysyłanie email.